scripts/versioningCheck.sh

```diff
@@ -1,9 +1,24 @@
 LATEST_RELEASE=`curl --silent "https://api.github.com/repos/joinColony/colonyNetwork/releases/latest" | grep -Po '"tag_name": "\K.*?(?=")'`
 
-NO_COMMENT_REGEX="^[ \t]*[^\/ \t][^\/ \t].*$"
+CURRENT_BRANCH=`git branch --show-current`
 
-# Are there changes in the colony contract?
-N=`git diff --cached --name-only -G"$NO_COMMENT_REGEX" $LATEST_RELEASE contracts/colony/ | wc -l`
+# Get release
+
+git checkout $LATEST_RELEASE
+
+# Compile release
+
+npm i --force && npx truffle compile
+rm -rf build-$LATEST_RELEASE || true
+mv build build-$LATEST_RELEASE
+
+git checkout yarn.lock
+rm package-lock.json
```

Comment:
Is there any handling if/when we move into a world where we're dealing only with the npm versions of the repo?